package.json

```diff
@@ -54,14 +54,16 @@
     "plugin-error": "^1.0.1",
     "prettier": "^1.16.4",
     "ts-node": "^7.0.1",
-    "tslib": "1.11.2",
```

Comment:
We can't really change this shared file and should revert the changes here

Reply:
reverted the file change